edge-functions/basic-auth-password/package.json

```diff
@@ -20,7 +20,7 @@
     "@types/react": "17.0.38",
     "autoprefixer": "^10.4.2",
     "eslint": "^8.9.0",
-    "eslint-config-next": "^12.1.0",
+    "eslint-config-next": "^12.1.7-canary.40",
```

Comment:
we've used `canary` and `
legacy-peer-deps=true ` inside npmrc in the other examples